src/components/CallView/CallView.vue

```diff
@@ -75,36 +75,14 @@
 							:shared-data="sharedDatas[shownRemoteScreenPeerId]"
 							is-big />
 						<!-- presenter overlay -->
-						<VueDraggableResizable v-if="shouldShowPresenterOverlay"
-							:key="presenterOverlaySize"
-							parent
-							:resizable="false"
-							:h="presenterOverlaySize"
-							:w="presenterOverlaySize"
-							:x="10"
-							:y="10">
-							<VideoVue class="presenter-overlay__video"
-								:token="token"
-								:model="shownRemoteScreenCallParticipantModel"
-								:shared-data="sharedDatas[shownRemoteScreenPeerId]"
-								is-presenter-overlay
-								un-selectable
-								hide-bottom-bar
-								@click-presenter="toggleShowPresenterOverlay" />
-						</VueDraggableResizable>
-						<!-- presenter button when presenter overlay is collapsed -->
-						<NcButton v-else-if="isPresenterCollapsed"
-							:aria-label="t('spreed', 'Show presenter')"
-							:title="t('spreed', 'Show presenter')"
-							class="presenter-overlay--collapse"
-							type="tertiary-no-background"
-							@click="toggleShowPresenterOverlay">
-							<template #icon>
-								<AccountBox fill-color="#ffffff" :size="20" />
-							</template>
-						</NcButton>
+						<PresenterOverlay v-if="shouldShowPresenterOverlay || isPresenterCollapsed"
+							:class="{'presenter-overlay--collapsed': isPresenterCollapsed }"
+							:token="token"
+							:model="shownRemoteScreenCallParticipantModel"
+							:shared-data="sharedDatas[shownRemoteScreenPeerId]"
+							:is-collapsed="isPresenterCollapsed"
```

Comment:
And use data directly
```suggestion
							:is-collapsed="!showPresenterOverlay"
```